aggregators/is-performant/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Aggregate = require('../aggregate');
```

Comment:
the distinction between aggregators and auditors right now isn't really clear to me either, both in where particular logic should live and why we need a distinction. For instance, why is the firstMeaningfulPaint audit assigning a score when that score will be changed by the aggregator? That leaves two places to make changes as we tweak the `is-performant` test to fit reality.

I was getting at this a bit in #61 about avoiding auditor auditors, but with this PR, it feels like we're trying to be too layered without much win for it, and we could just be doing something like make the aggregators audits and the existing audits would just be function calls made by the them.